docs/userguide/ui-concepts.md

```diff
@@ -1,3 +1,150 @@
 # Workflow UI
 
-_Coming soon!_
+This page provides a high level overview of Workflow UI, the companion that allows [Workflow Core](../core-workflow) to drive Android and iOS apps.
+
+## Separation of Concerns
+
+Workflow maintains a rigid separation between its core runtime and its UI support.
+The [Workflow Core](../core-workflow) modules are strictly Swift and Kotlin, with no dependencies on any UI framework.
+Dependencies on Android and iOS are restricted to the Workflow UI modules, as you would expect.
+This innate separation naturally puts developers on a path to avoid entangling view concerns with their app logic.
+
+And note that we say "app logic" rather than "business logic."
+In any interesting app, the code that manages navigation and other UI-releated behavior is likely to dwarf that for what we typically think of as model concerns, in both size and complexity.
+
+We're all pretty good at capturing business concerns in tidy OO models of items for sale, shopping carts, payment cards and the like, nicely decoupled from the UI world.
```

Comment:
When I read this, I felt that 'OO' kind of interrupted the flow.

```suggestion
We're all pretty good at capturing business concerns in tidy object-oriented models of items for sale, shopping carts, payment cards and the like, nicely decoupled from the UI world.
```

Reply:
Same, good change I think.

Reply:
Nice, will do this in the next update.